abci/vote_extensions.go

```diff
@@ -71,35 +72,59 @@ func (h *VoteExtensionHandler) ExtendVoteHandler() sdk.ExtendVoteHandler {
 	return func(ctx sdk.Context, req *cometabci.RequestExtendVote) (*cometabci.ResponseExtendVote, error) {
 		// Iterate through auction bids until we find a valid one
 		auctionIterator := h.tobLane.Select(ctx, nil)
+		txsToRemove := make(map[sdk.Tx]struct{}, 0)
 
 		for ; auctionIterator != nil; auctionIterator = auctionIterator.Next() {
 			bidTx := auctionIterator.Tx()
 
 			// Verify the bid tx can be encoded and included in vote extension
-			if bidBz, err := h.txEncoder(bidTx); err == nil {
```

Comment:
Why change this logic?

Reply:
solely for logging purposes. its easier to log the errors this way